extensions/amp-lightbox-gallery/1.0/amp-lightbox-gallery.js

```diff
@@ -50,6 +50,23 @@ class AmpLightboxGallery extends BaseElement {
     );
     return super.isLayoutSupported(layout);
   }
+
+  /** @override */
+  afterOpen() {
+    const scroller = this.element.shadowRoot.querySelector('[part=scroller]');
```

Comment:
nit: even though it's empty, let's call the super method for consistency
```suggestion
    super.afterOpen();
    const scroller = this.element.shadowRoot.querySelector('[part=scroller]');
```